### DISCUSSION TITLE
Set the default style in C#?

### DISCUSSION THREAD

**Discussion by Mtax-Development** [AvaloniaUI/Avalonia#8472] [Q&A] [answered]
I am attempting to use one of default styles through C# using the `Application` class. In XAML this is done with the following:
```
<Application>
	<Application.Styles>
		<FluentTheme Mode="Light"/>
	</Application.Styles>
</Application>
```
What would be the equivalent code in C#?

**Comment by maxkatz6** [comment, marked as answer]
It will be the same, just in C# syntax:
```csharp
Application.Styles.Add(new FluentTheme { Mode = "Light" } )
```
Where Application is instance of your app (`this` in the application class or `Application.Current` if globally)

**Reply by Mtax-Development** [reply]
`FluentTheme` requires use of a XAML file as a construction argument. Is there a way to construct it without arguments like you are suggesting that I am missing?

**Reply by maxkatz6** [reply]
It needs a Uri object. This uri is used to resolve relative paths in xaml files. You should pass name of your assembly most of the time, but for FluentTheme is doesn't matter.
`new FluentTheme(new Uri("avares://MyAssembly")) {}`
